packages/create-cloudflare/src/helpers/__tests__/command.test.ts

```diff
@@ -57,17 +56,8 @@ describe("Command Helpers", () => {
 		}));
 	});
 
-	const expectSpawnWith = (cmd: string) => {
-		const [command, ...args] = shellquote.parse(cmd);
-
-		expect(spawn).toHaveBeenCalledWith(command, args, {
-			stdio: "inherit",
-			env: process.env,
-		});
-	};
-
 	const expectSilentSpawnWith = (cmd: string) => {
-		const [command, ...args] = shellquote.parse(cmd);
+		const [command, ...args] = cmd.split(" ");
```

Comment:
(Related to my more general comment https://github.com/cloudflare/workers-sdk/pull/4445#issuecomment-1814872733)

This is an example of where we still end up using `.split(" ")` – if `cmd` has an values with spaces, this will parse incorrectly

Reply:
This is a test helper and purely there as a convenience. The function is only used in this file and we can see that there are no complex args. If you feel strongly about this then we can also split those args manually in the callers here.